matplotlibrc.template

```diff
@@ -279,6 +280,22 @@ backend      : %(backend)s
 #polaraxes.grid      : True    # display grid on polar axes
 #axes3d.grid         : True    # display grid on 3d axes
 
+### DATES
+# These control the default format strings used in AutoDateFormatter.
+# Any valid format datetime format string can be used (see the python
+# `datetime` for details).  For example using '%%x' will use the locale date representation
+# '%%X' will use the locale time representation and '%%c' will use the full locale datetime
+# representation.
+# These values map to the scales:
```

Comment:
And actually it's only needed for the template.  People's local matplotlibrc's don't need it.  This is because the template is first interpolated to set the default backend.

We could use `str.format` (or some other method for replacing a string) to get around this, which might be less confusing.